Core/OfficeDevPnP.Core/packages.config

```diff
@@ -7,6 +7,7 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
   <package id="Microsoft.Graph" version="1.1.1" targetFramework="net45" />
   <package id="Microsoft.Graph.Core" version="1.2.1" targetFramework="net45" />
+  <package id="Microsoft.Identity.Model.Extensions" version="2.0.1459.0" targetFramework="net45" />
```

Comment:
I've removed that reference. I must be missing something...without that Nuget reference, I can't build the solution. I've got the Windows Identity Foundation installed, so I'm not sure what I'm missing. Ideas?